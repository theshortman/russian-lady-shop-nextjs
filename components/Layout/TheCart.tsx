import Link from "next/link";
import { FC } from "react";

const TheCart: FC = () => {
  return (
    <div className="list-group mb-3">
      <Link href="/cart">
        <a className="list-group-item text-reset">
          <div className="list-group-item-heading">
            <span className="text-danger">КОРЗИНА</span>
          </div>

          <div className="list-group-item-text">Пока пусто!</div>
        </a>
      </Link>
    </div>
  );
};

export default TheCart;
