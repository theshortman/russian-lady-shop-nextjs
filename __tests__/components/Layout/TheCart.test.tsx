import { render } from "@testing-library/react";
import TheCart from "@/components/Layout/TheCart";

test("renders TheCart unchanged", () => {
  const { container } = render(<TheCart />);
  expect(container).toMatchSnapshot();
});
