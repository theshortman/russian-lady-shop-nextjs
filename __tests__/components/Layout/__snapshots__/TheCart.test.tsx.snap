// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TheCart unchanged 1`] = `
<div>
  <div
    class="list-group mb-3"
  >
    <a
      class="list-group-item text-reset"
      href="/cart"
    >
      <div
        class="list-group-item-heading"
      >
        <span
          class="text-danger"
        >
          КОРЗИНА
        </span>
      </div>
      <div
        class="list-group-item-text"
      >
        Пока пусто!
      </div>
    </a>
  </div>
</div>
`;
